plugins/a11yhelp/plugin.js

```diff
@@ -20,32 +20,190 @@
 		availableLangs: { af:1,ar:1,az:1,bg:1,ca:1,cs:1,cy:1,da:1,de:1,'de-ch':1,el:1,en:1,'en-gb':1,eo:1,es:1,'es-mx':1,et:1,eu:1,fa:1,fi:1,fo:1,fr:1,'fr-ca':1,gl:1,gu:1,he:1,hi:1,hr:1,hu:1,id:1,it:1,ja:1,km:1,ko:1,ku:1,lt:1,lv:1,mk:1,mn:1,nb:1,nl:1,no:1,oc:1,pl:1,pt:1,'pt-br':1,ro:1,ru:1,si:1,sk:1,sl:1,sq:1,sr:1,'sr-latn':1,sv:1,th:1,tr:1,tt:1,ug:1,uk:1,vi:1,zh:1,'zh-cn':1 },
 		// jscs:enable
 
+		onLoad: function() {
+			CKEDITOR.document.appendStyleSheet( CKEDITOR.getUrl( this.path + 'styles/a11yhelp.css' ) );
+		},
+
 		init: function( editor ) {
 			var plugin = this;
-			editor.addCommand( commandName, {
-				exec: function() {
-					var langCode = editor.langCode;
-					langCode =
-						plugin.availableLangs[ langCode ] ? langCode :
-						plugin.availableLangs[ langCode.replace( /-.*/, '' ) ] ? langCode.replace( /-.*/, '' ) :
-						'en';
-
-					CKEDITOR.scriptLoader.load( CKEDITOR.getUrl( plugin.path + 'dialogs/lang/' + langCode + '.js' ), function() {
-						editor.lang.a11yhelp = plugin.langEntries[ langCode ];
-						editor.openDialog( commandName );
-					} );
-				},
-				modes: { wysiwyg: 1, source: 1 },
-				readOnly: 1,
-				canUndo: false
-			} );
+			var langCode = editor.langCode;
+			langCode =
+				plugin.availableLangs[ langCode ] ? langCode :
+				plugin.availableLangs[ langCode.replace( /-.*/, '' ) ] ? langCode.replace( /-.*/, '' ) :
+				'en';
+			CKEDITOR.scriptLoader.load( CKEDITOR.getUrl( plugin.path + 'dialogs/lang/' + langCode + '.js' ), function() {
+				editor.lang.a11yhelp = plugin.langEntries[ langCode ];
+
+				editor.addCommand( commandName, new CKEDITOR.dialogCommand( commandName, {
+					modes: { wysiwyg: 1, source: 1 },
+					readOnly: 1,
+					canUndo: false,
+					label: editor.lang.a11yhelp.accessibilityCommandsLabels.generalTab
+				} ) );
+
+				editor.addCommand( 'keystrokesTab', new CKEDITOR.dialogCommand( commandName, {
+					modes: { wysiwyg: 1, source: 1 },
+					readOnly: 1,
+					canUndo: false,
+					label: editor.lang.a11yhelp.accessibilityCommandsLabels.keystrokesTab,
+					tab: 'keystrokes'
+				} ) );
 
-			editor.setKeystroke( CKEDITOR.ALT + 48 /*0*/, 'a11yHelp' );
-			CKEDITOR.dialog.add( commandName, this.path + 'dialogs/a11yhelp.js' );
+				editor.setKeystroke( CKEDITOR.ALT + 48 /*0*/, 'a11yHelp' );
+				CKEDITOR.dialog.add( commandName, plugin.path + 'dialogs/a11yhelp.js' );
+
+				editor.setKeystroke( CKEDITOR.CTRL + 191 /*/*/, 'keystrokesTab' );
+				CKEDITOR.dialog.add( 'keystrokesTab', plugin.path + 'dialogs/a11yhelp.js' );
```

Comment:
I didn't know that. I've base on find plugin:
https://github.com/ckeditor/ckeditor-dev/blob/2249f1a3172f98d62cc404f2b1ab4c9018d427dc/plugins/find/plugin.js#L35-L36
Which opens on different tab when is 'find' or 'replace' used.

Reply:
It's a slightly different thing, as in fact it does define two separate dialogs: https://github.com/ckeditor/ckeditor-dev/blob/2249f1a3172f98d62cc404f2b1ab4c9018d427dc/plugins/find/dialogs/find.js#L817-L823

And it does so because... someone was lazy enough not to do what you made in #830. 🙂 

Actually having #830 in place we can refactor it, I will create a proper ticket for that in a minute.